cmd/signals.go

```diff
@@ -78,18 +78,18 @@ func handleSignals() {
 		case <-globalHTTPServerErrorCh:
 			exit(stopProcess())
 		case osSignal := <-globalOSSignalCh:
-			logger.Info("Exiting on signal: %s", strings.ToUpper(osSignal.String()))
+			logStartupMessage("Exiting on signal: %s", strings.ToUpper(osSignal.String()))
```

Comment:
Since this is not really a `startup` message, is it better to have a more generic name for the function, say `logInfoMessage`?

Reply:
It is start-up error since I am repurposing the same code, I didn't want to change the name significantly to avoid too many line changes.

Reply:
But I think changing the name is important here.